src/routes/dashboard/fragment-tile.tsx

```diff
@@ -105,53 +83,17 @@ export const FragmentTile = ({
 		});
 	};
 
-	const renderProps: RenderProps = {
-		id: fragment.id,
-		name: fragment.title,
-		width: 800,
-		height: 400,
-		preview: {
-			format: 'png',
-			width: 330,
-			height: 200
-		}
-	};
-
 	const resetPreview = async () => {
-		const imageBlob = await getFragmentPreview(fragment, renderProps);
-		const reader = new FileReader();
-		reader.readAsDataURL(imageBlob ? imageBlob : new Blob());
-		reader.onloadend = () => {
-			const imageUrl: string = reader.result ? reader.result.toString() : '';
-			setPreviewUrl(imageUrl);
-		};
+		const url = await getPreviewUrl(fragment);
+		setPreviewUrl((url as string));
```

Comment:
```suggestion
		setPreviewUrl(url as string);
```
?

Reply:
or just

```suggestion
		setPreviewUrl(url);
```

if `getPreviewUrl` resolves to string, which it seems it does.

Reply:
maybe even oneline it if we don't do any checks?

Reply:
It still complains if I don't include the as string, I can oneline it though